### ISSUE TITLE
Cuda driver error in training on perlmutter

### ISSUE THREAD

**Issue by rebassoo** [ai2cm/ace#504]
When training with a recent image as of 56312e69 ran into driver error:

`RuntimeError: handle_0 INTERNAL ASSERT FAILED at "/opt/pytorch/pytorch/c10/cuda/driver_api.cpp":15, please report a bug to PyTorch`

See full log here:
[33272732.txt](https://github.com/user-attachments/files/17909222/33272732.txt)

**Comment by oliverwm1** [comment]
Thanks for reporting. And to confirm—you can use the exact same dataset/config, with a different code version, and get training to work? And are you using podman-hpc on perlmutter, or using a conda environment?

**Comment by rebassoo** [comment]
Yes I can use the same dataset/config with a different code version (for instance fme:e2382139) and get training to work. I am using podman-hpc.

**Comment by oliverwm1** [comment]
Training has been working on perlmutter lately.
